package.json

```diff
@@ -551,14 +551,9 @@
                 "when": "listFocus && !inputFocus"
             },
             {
-                "key": "/",
-                "command": "list.toggleKeyboardNavigation",
-                "when": "listFocus && !inputFocus && listSupportsKeyboardNavigation"
-            },
-            {
-                "key": "escape",
-                "command": "list.toggleKeyboardNavigation",
-                "when": "listFocus && inputFocus && listSupportsKeyboardNavigation && !notebookCellListFocused"
+                "key": "oem_2",
```

Comment:
Why do you use `oem_2`? Why not `/` for readibility?
Also please re-add `!inputFocus` as it can interfere with file renaming in explorer.